src/lib/__tests__/apis/fetch.test.ts

```diff
@@ -40,7 +40,7 @@ it("tries to parse the response when there is a String and resolves with it (2)"
     callback(null, reqResponse)
   })
 
-  return expectPromiseRejectionToMatch(fetch("foo/bar"), /Unexpected token/)
+  return expectPromiseRejectionToMatch(fetch("foo/bar"), /(Expected property)/)
```

Comment:
This appears to be a change in `node`'s json parser error message, but trying to get more confirmation.

Reply:
Interesting. What the new error message says is completely the opposite of the previous one.

Reply:
Sorry, should have been more specific here. I am struggling to find the docs that say it explicitly, but my understanding is that at some point the `v8` engine that `node` is built on, and what implements `JSON.parse`, updated their error implementation to have more descriptive error messages. So with the node bump, we also received a bump in the `v8` engine. 

So the full error would be something like `"Expected property name or '}' in JSON at position 2 (line 1 column 3)"`.